unidist/core/backends/mpi/core/executor.py

```diff
@@ -536,16 +537,18 @@ def push_data(dest_rank, value):
 # Control API
 # -----------------------------------------------------------------------------
 
-# TODO: implement
+
 def init():
     """
     Initialize MPI processes.
 
     Notes
     -----
-    Currently does nothing.
+    Only collects the MPI cluster topology.
     """
-    pass
+    global topology
+    if len(topology) == 0:
```

Comment:
?
```suggestion
    if not topology:
```

Reply:
I would prefer to be more concrete here and check the presence of elements in `topology`.

Reply:
I thought it is the same syntax

Reply:
Applied your suggestion. This way is aligned with pep https://docs.python.org/3/library/stdtypes.html#truth-value-testing